lib/logic/models/sort.dart

```diff
@@ -1,15 +1,19 @@
-import 'package:enum_to_string/enum_to_string.dart';
 import 'package:equatable/equatable.dart';
 import 'package:sweyer/sweyer.dart';
 
-/// Interface for other sort feature enums.
-abstract class SortFeature<T extends Content> extends Enum<String> {
-  const SortFeature._(String value) : super(value);
+/// The order of a sort operation.
+enum SortOrder {
+  /// Ascending sort order ("lower" values first, "higher" values last).
+  ascending,
+
+  /// Descending sort order ("higher" values first, "lower" values last).
+  descending,
+}
 
+/// Interface for other sort feature enums.
+abstract class SortFeature<T extends Content> {
   /// Returns sort feature values for a given content.
   static List<SortFeature<T>> getValuesForContent<T extends Content>(ContentType<T> contentType) {
-    // TODO: Remove ContentType cast, see https://github.com/dart-lang/language/issues/2315
```

Comment:
Because https://github.com/dart-lang/language/issues/2315 has gotten no traction in the last two years, and since with the new Flutter update there is no longer a lint against this, I think it's acceptable to leave the cast. The original intent of the todo was to be able to get rid of the lint ignore below.